project.template.ts

```diff
@@ -88,7 +88,6 @@ const project: SubstrateProject = {
     {
       kind: SubstrateDatasourceKind.Runtime,
       startBlock: 1,
-      endBlock: 1,
```

Comment:
`endBlock` was used to make sure the genesis migration is run only once. 
I used the reference from here(https://academy.subquery.network/faqs/faqs.html#inserting-seed-data-at-project-initiation) to add this in.